ci-operator/config/openshift/cluster-api-actuator-pkg/openshift-cluster-api-actuator-pkg-release-4.18__images.yaml

```diff
@@ -0,0 +1,28 @@
+base_images:
+  openshift_release_golang-1.21:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: Dockerfile.ci
```

Comment:
Thanks Zhaohua for initiating the effort. Would you create a separate qe customized Dockerfile later?

Reply:
Do we have a need to create a customised image? The current one copies the entire source in, so, we will have access to both QE and dev tests from there

Reply:
This image is only used for testing, it doesn't ship as part of the product so I think I'm ok to keep them together even if we need to add new tools

Reply:
ah, that would be great! Thank you.